.github/workflows/continuous-delivery.yml

```
@@ -0,0 +1,140 @@
+          POSTGRES_USER: oggole
+          POSTGRES_PASSWORD: oggole
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)